README.md

```diff
@@ -11,10 +11,21 @@ A collection of reusable ansible components used by the EthPandaOps team.
 - [ethereum_auth_jwt](roles/ethereum_auth_jwt)
 - [dshackle](roles/dshackle)
 
-### Ethereum clients
+### Ethereum client pair
+- [ethereum_node](roles/ethereum_node)
+### Ethereum execution clients
+- [besu](roles/besu)
 - [geth](roles/geth)
+- [erigon](roles/erigon)
+- [nethermind](roles/nethermind)
+### Ethereum consensus clients
 - [teku](roles/teku)
 - [nimbus](roles/nimbus)
+- [prysm](roles/prysm)
```

Comment:
Would we then extend this to add another `validator` role or would that be turned on/off based on a `enable_validator` flag?